### PR TITLE
Remove Red Hat branding and added downstream docs

### PR DESCRIPTION
**Pull request by ritz303**
## Description
Remove Red Hat branding from documentation and ported over the downstream content.

Fixes: Jira [DBAAS-1232](https://issues.redhat.com/browse/DBAAS-1232)

## Type of change
- [ ] Content edits to the YAML QSG files.
- [ ] Ported over the downstream Quick Start Guide and Reference Guide.

## Checklist (Definition of Done)

- [ ] All acceptance criteria specified in JIRA or in issue number have been completed
- [ ] Unit tests added that prove the fix is effective or the feature works 
- [ ] Documentation added for the feature
- [ ] all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer

### CODE REVIEW

File: controllers/reconcilers/quickstartinstallation/connecting-an-application-to-a-database-instance-using-the-topology-view-quick-start.yaml [hunk 1]
```diff
@@ -10,12 +10,12 @@ metadata:
...
   description: 'Use the Topology view to connect database instances to your applications.'
-  prerequisites: ["Access to the OpenShift console.", "Installation of the Red Hat OpenShift Database Access add-on.", "A cloud-hosted database provider added to your project."]
+  prerequisites: ["Access to the OpenShift console.", "Installation of the OpenShift Database Access add-on.", "A cloud-hosted database provider added to your project."]
```
Comment (mrrajan) [inline controllers/reconcilers/quickstartinstallation/connecting-an-application-to-a-database-instance-using-the-topology-view-quick-start.yaml:18]: Please update the "add-on" to "operator"

Comment (ritz303) [reply controllers/reconcilers/quickstartinstallation/connecting-an-application-to-a-database-instance-using-the-topology-view-quick-start.yaml]: @mrrajan : Good catch! Thanks for taking a look.

**Review by tchughesiv — CHANGES_REQUESTED** [review]
we may have to move away from using mongo as the example, depending on what they decide wrt merging our code upstream. maybe the safer route is to use RDS since we're more likely to package that with our release?

**Review by tchughesiv — CHANGES_REQUESTED** [review]
also, could you please remove any references you find around this being a "service preview" release... or beta release? thanks

File: controllers/reconcilers/quickstartinstallation/installing-the-openshift-database-access-operator-quick-start.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  description: 'Install the OpenShift Database Access operator in your cluster.'
+  prerequisites: ["OpenShift Container Platform 4.10 or higher.", "Running a OpenShift Dedicated, or an OpenShift Service on Amazon Web Services (AWS).", "AWS account and credentials."]
```
[the remark this answers is not in the document]
Comment (ritz303) [reply controllers/reconcilers/quickstartinstallation/installing-the-openshift-database-access-operator-quick-start.yaml]: @tchughesiv : Thanks for the feedback!  I've updated the prereqs accordingly.

File: controllers/reconcilers/quickstartinstallation/installing-the-openshift-database-access-operator-quick-start.yaml [hunk 3]
```diff
@@ -15,7 +15,7 @@ spec:
...
   description: 'Install the OpenShift Database Access operator in your cluster.'
-  prerequisites: ["OpenShift Container Platform 4.10 or higher.", "Running a OpenShift Dedicated, or an OpenShift Service on Amazon Web Services (AWS).", "AWS account and credentials."]
+  prerequisites: ["Either OpenShift Container Platform or Origin Key Distribution (OKD) 4.10 or higher"]
```
Comment (tchughesiv) [inline controllers/reconcilers/quickstartinstallation/installing-the-openshift-database-access-operator-quick-start.yaml:18]: Is that what OKD stands for? I always thought it was OpenShift Community Distribution

**Review by tchughesiv — CHANGES_REQUESTED** [review]
@ritz303 i think we'll need to remove mentions of mongodb atlas from the docs. let's use RDS as an example instead and in docs/references

also, please remove mentions of this release being "service preview". tks!

**Comment by ritz303** [comment]
@tchughesiv :  I thought I got all of the "Service Preview" instances, but I did find one more, so thank you for double checking me.  I've also replaced all mentions of MongoDB and replaced them with Amazon RDS.  Please let me know if I've missed anything.

**Comment by ritz303** [comment]
@tchughesiv : Thanks for taking a look!  I've removed the draft status.  I'm curious, can you do a squash on my commits before merging this PR into main?

**Review by ritz303 — COMMENTED** [review]
Finished reviewing the documentation.

**Review by tchughesiv — APPROVED** [review]

**Pull request merged**